discount/src/test/java/com/kyriba/conference/discount/DiscountControllerTest.java

```diff
@@ -30,6 +35,9 @@
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
 public class DiscountControllerTest {
 
+    @MockBean
+    private DiscountService discountService;
```

Comment:
For API level tests real DB is used, for example with testcontainers.

Reply:
Ok